src/web_interface/templates/generic_view/firmware_detail_tabular_field.html

```diff
@@ -1,12 +1,12 @@
 <li class="list-group-item p-2 clickable li-hover" onclick="location.href='/analysis/{{ firmware[0] }}'">
-    <a href='/analysis/{{ firmware[0] }}' style="color: rgb(51, 51, 51);">
-        <div class="d-flex justify-content-between align-items-center" style="width: 100%; height: 100%; font-size: 14px">
+    <a href='/analysis/{{ firmware[0] }}'>
+        <div class="d-flex justify-content-between align-items-center" style="width: 100%; height: 100%; font-size: 14px; color: #515151;">
             {{ firmware[1] }}
             <span>{{ firmware[2] | render_tags(size=11) | safe}}</span>
         </div>
     </a>
     <div class="d-flex justify-content-between align-items-center">
-        <span style="font-family: 'Courier New', Courier, monospace; font-size: 10px">{{ firmware[3] | nice_unix_time | safe }}</span>
+        <span class="footnote text-muted">{{ firmware[3] | nice_unix_time | safe }}</span>
     	<a href="/download/{{ firmware[0] }}" class="pull-right"><i class="fas fa-download"></i></a>
```

Comment:
Does this work as expected? From the bootstrap 3 to 4 migration doc:

> Added .float-{sm,md,lg,xl}-{left,right,none} classes for responsive floats and removed .pull-left and .pull-right since they’re redundant to .float-left and .float-right.

Reply:
what do you mean? "footnote" is defined in `home.html` and only contains what was  in `style="font-family: 'Courier New', Courier, monospace; font-size: 10px`

Reply:
pull-right

Reply:
Not your changes